docs/reference/api/storage/FlashSimBlockDevice.md

```diff
@@ -0,0 +1,28 @@
+## FlashSimBlockDevice
+
+<span class="images">![](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/class_slicing_block_device.png)<span>FlashSimBlockDevice class hierarchy</span></span>
```

Comment:
Note: We have to replace this with the diagram for FlashSimBlockDevice once the code merges.